mypy/mypyc_hacks.py

```diff
@@ -1,24 +1,26 @@
 """Stuff that we had to move out of its right place because of mypyc limitations."""
 
-from typing import Dict, Any
+from typing import Dict, Any, TextIO
 import sys
 
 
 # Extracted from build.py because we can't handle *args righit
 class BuildManagerBase:
-    def __init__(self) -> None:
+    def __init__(self, stdout: TextIO, stderr: TextIO) -> None:
```

Comment:
I like how here it's a mandatory argument, no default, so no premature locking in of `sys.stdout/stderr`.